checkstyle/checkstyle.xml

```diff
@@ -64,6 +64,9 @@
     <module name="ParameterName"/>
     <module name="StaticVariableName"/>
     <module name="TypeName"/>
+    <module name="AvoidStarImport">
+      <property name="allowClassImports" value="false"/>
```

Comment:
Is this line doing anything? From the checkstyle docs it looks like it's the default? Would a single `<module name="AvoidStarImport" />` line here be enough?